### PR TITLE
Add AppCleaner Extension

### PR DESCRIPTION
**Pull request by yuhixyz**
## Description

Uninstall Application by AppCleaner.

## Screencast

image

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed in the `metadata` folder

### CODE REVIEW

**Comment by anhthang** [comment]
Raycast already have this in v1.31.0

https://www.raycast.com/changelog

**Comment by yuhixyz** [comment]
> Raycast already have this in v1.31.0
> 
> https://www.raycast.com/changelog
I noticed that. But I found that not every app can be uninstalled in that way. For example,when I search kitty and open the actionpanel, there's no uninstall option.

File: extensions/application-uninstaller/package.json [hunk 1]
```diff
@@ -0,0 +1,43 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "application-uninstaller",
+  "title": "Application Uninstaller",
```
Comment (pernielsentikaer) [inline extensions/application-uninstaller/package.json:4]:
```suggestion
  "title": "AppCleaner Uninstaller",
```

Comment (pernielsentikaer) [reply extensions/application-uninstaller/package.json:4]: Could be a better name, could you also rename the extension folder?

File: extensions/application-uninstaller/src/index.tsx [hunk 2]
```diff
@@ -0,0 +1,56 @@
+import {
+  List,
+  ActionPanel,
+  ActionPanelItem,
```
Comment (pernielsentikaer) [inline extensions/application-uninstaller/src/index.tsx:4]:
```suggestion
  Action,
```
Deprecated, use Action instead

File: extensions/application-uninstaller/src/index.tsx [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+  Application,
+  getApplications,
+  closeMainWindow,
+  popToRoot,
+} from "@raycast/api";
+import { useState, useEffect } from "react";
+import { execaCommand } from "execa";
```
Comment (pernielsentikaer) [inline extensions/application-uninstaller/src/index.tsx:11]:
```suggestion
```
Declared but never used

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi @yuhixyz 👋 

Thanks for your contribution 🔥

I have some feedback and suggestions for you to address:

- Could you add a [screenshot](https://developers.raycast.com/basics/prepare-an-extension-for-store#screenshots) and [changelog](https://developers.raycast.com/basics/prepare-an-extension-for-store#version-history)

- You could add a check if AppCleaner is installed, something like it's [done here](https://github.com/raycast/extensions/tree/main/extensions/flow)

Looking forward to testing it out again 💪 

Request a new review when you are ready, feel free to contact me here or at [Slack](https://raycastcommunity.slack.com/) if you have any questions.

**Comment by koinzhang** [comment]
> > Raycast already have this in v1.31.0
> > [raycast.com/changelog](https://www.raycast.com/changelog)
> 
> I noticed that. But I found that not every app can be uninstalled in that way. For example,when I search kitty and open the actionpanel, there's no uninstall option.

Hi, @yuhixyz. This is not a bug, the uninstall option does not appear in the actionPanel when the application is running.
When the app is running, the app icon is marked with a small dot below it. Try uninstalling using Raycast after exiting kitty.

**Comment by yuhixyz** [comment]
> 



> > > Raycast already have this in v1.31.0
> > > [raycast.com/changelog](https://www.raycast.com/changelog)
> > 
> > 
> > I noticed that. But I found that not every app can be uninstalled in that way. For example,when I search kitty and open the actionpanel, there's no uninstall option.
> 
> Hi, @yuhixyz. This is not a bug, the uninstall option does not appear in the actionPanel when the application is running. When the app is running, the app icon is marked with a small dot below it. Try uninstalling kitty by Raycast after exiting it.

@koinzhang Thanks a lot. I didn't realize that kitty is running. So this extension is no longer needed I think. I'll close this pr.

**Pull request closed without merging**